src/mono/wasm/test-main.js

```diff
@@ -63,18 +63,21 @@ async function getArgs() {
         queryArguments = Array.from(WScript.Arguments);
     }
 
-    let runArgs;
-    if (queryArguments.length > 0) {
-        runArgs = processArguments(queryArguments);
-    } else {
-        const response = fetch('/runArgs.json')
-        if (!response.ok) {
-            console.debug(`could not load /args.json: ${response.status}. Ignoring`);
+    let runArgsJson;
+    // ToDo: runArgs should be read for all kinds of hosts, but
```

Comment:
Can you please open an issue to track this?

Reply:
https://github.com/dotnet/runtime/issues/81416